cycledash/api/ga4gh_wrapper.py

```diff
@@ -0,0 +1,47 @@
+"""A subset of the GA4GH API for use in Cycledash."""
```

Comment:
Minor nit, but maybe rename this file `ga4gh.py`? (Since it's in `/api/`, everything within is pretty much a wrapper.)